enki/PhysicalEngine.cpp

```diff
@@ -1145,16 +1145,25 @@ namespace Enki
 		{
 			// init physics interactions
 			for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
-				(*i)->initPhysicsInteractions(overSampledDt);
+			{
+				if(SkipPhysicsObjectsList.find(*i) == SkipPhysicsObjectsList.end())
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.